.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,9 @@
 > If you do not have an issue number, please have a good description of
 > the problem and the fix. Help the reviewer understand what to expect.
 >
+> Add a `### Testing` section to your PR to describe how testing was done.
```

Comment:
This should just be an actual section above these comments or something, so the PR creator does not have to go type "### Testing" all the time.